### ISSUE TITLE
clean up text around dates

### ISSUE THREAD

**Issue by sastels** [cds-snc/report-a-cybercrime#809]
clarify better that can be vague / missing / etc

**Comment by sastels** [comment]
Redesigned in new pilot.
